### DISCUSSION TITLE
*PWA banned from iOS in Europe:*

### DISCUSSION THREAD

**Discussion by andreaswittmann** [200ok-ch/organice#979] [Q&A]
Heise reports that /Progressive Web Applications/ will be banned from iOS Devices in Europe, starting with release iOS 17.4 in March 2024.
https://www.heise.de/news/Apple-bestaetigt-Keine-Progressive-Web-Apps-mehr-fuer-iOS-in-der-EU-9630089.html
What implication will this have for iPhone Users?

**Comment by munen** [comment]
Thank you for the heads-up. I'm running 17.3.1, so I'll make sure not to upgrade and see what kind of fallout happens. The whole situation sounds like a "self harm as a threat" tactic. Let's see how that goes for them. 

In the long run, it might just turn out strongly beneficial with proper alternative browsers coming to iOS. The PWA story was pretty weak on iOS, anyway.

**Comment by munen** [comment]
All clear: https://9to5mac.com/2024/03/01/apple-home-screen-web-apps-ios-17-eu/
